init_node_script_test.go

```
@@ -93,31 +96,33 @@ printf '%s\n' "$*" >> "$INVOCATIONS"
   mkdir -p "$HOME/.truerepublic/config"
   printf '{}\n' > "$HOME/.truerepublic/config/genesis.json"
+  printf 'prometheus = false\nprometheus_listen_addr = ":26660"\n' > "$HOME/.truerepublic/config/config.toml"
 fi
 `
 	if err := os.WriteFile(fakeBinary, []byte(fake), 0o700); err != nil {
 		t.Fatal(err)
 	}
 
-	run := func(operator string) ([]byte, error) {
+	run := func(operator, prometheus string) ([]byte, error) {
 		cmd := exec.Command("sh", "scripts/docker-entrypoint.sh", "start")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)